tests/test_sequence.py

```diff
@@ -91,26 +91,24 @@ def test_increment_wraps_image_sequence(self):
             expected_seq_no = i % sequence_range
             self.mock_sequence_response(self.image_sequence_endpoint, i, sequence_start, sequence_range, expected_sequence_no=expected_seq_no)
 
+    def test_increments_json_sequence(self):
+        sequence_start = 1
+        sequence_range = 1000000000
 
-def test_increments_json_sequence(self):
-    sequence_start = 1
-    sequence_range = 1000000000
-
-    test_start = 1
-    test_end = 10
-
-    for i in range(test_start, test_end):
-        expected_seq_no = i % sequence_range
-        self.mock_sequence_response(self.json_sequence_endpoint, i, sequence_start, sequence_range, expected_sequence_no=expected_seq_no)
+        test_start = 1
+        test_end = 10
```

Comment:
Why has this changed? Is it just formatting? If so, what was wrong with it?

Reply:
These weren't in the `TestSequenceService` class and they were failing, but passed when moved in